src/vaadin-date-time-picker.html

```diff
@@ -263,6 +263,11 @@
               value: ''
             },
 
+            /**
+             * Set true to prevent the overlay from opening automatically.
```

Comment:
I think it would be nice if this also explained in which cases the overlays do open automatically by default. Maybe not a blocker, but at least I would change the current sentence to "Set __to__ true to prevent the __overlays__ from opening automatically." (looks weird without the "to" and note there are 2 overlays one for date and one for time).

Also I'm wondering would it make sense to enable setting this separately only for date or time picker? Maybe the user would want to only disable the auto open for one or the other? It could be handled in the same way as `placeholder`. For that in this component we have `datePlaceholder` and `timePlaceholder`, so we could have `dateAutoOpenDisabled` and `timeAutoOpenDisabled`. Though to me it feels like maybe this is enough for now to only have one property that is used for both and it can be changed later if it becomes a requested feature.

Reply:
ping

Reply:
@tulioag Since you're updating this, see my suggestion about using this instead "Set to true to prevent the overlays from opening automatically." (you can ignore the rest of my comment above).

Reply:
Done